src/flowr.ts

```diff
@@ -26,7 +26,7 @@ import { flowrVersion } from '@eagleoutice/flowr/util/version'
 
 const scriptsText = Array.from(Object.entries(scripts).filter(([, { type }]) => type === 'master script'), ([k,]) => k).join(', ')
 // this is automatically replaced with the current version by release-it
-const cliVersion = '0.0.3'
+export const cliVersion = '0.0.3'
```

Comment:
```suggestion
export const CLI_VERSION = '0.0.3'
```
Just so that we make the variables nature clearer?

Reply:
we'll have to change it in the package.json too, but yea